.editorconfig

```diff
@@ -80,3 +80,207 @@ dotnet_naming_rule.private_const_fields_ignore_camel_case_leading_underscore.sev
 dotnet_naming_rule.private_const_fields_ignore_camel_case_leading_underscore.symbols = private_const_fields
 dotnet_naming_rule.private_const_fields_ignore_camel_case_leading_underscore.style = camel_case_leading_underscore
 
+# StyleCop
+# Rules are listed at https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/rulesets/StyleCopAnalyzersDefault.ruleset
+
+# StyleCop: Spacing Rules
+[*.cs]
+dotnet_diagnostic.SA1000.severity = none 			# Keywords should be spaced correctly.
+dotnet_diagnostic.SA1001.severity = none			# Commas should be spaced correctly.
+dotnet_diagnostic.SA1002.severity = none			# Semicolons should be spaced correctly.
+dotnet_diagnostic.SA1003.severity = none			# Symbols should be spaced correctly.
+dotnet_diagnostic.SA1004.severity = none			# Documentation lines should begin with single space.
+dotnet_diagnostic.SA1005.severity = none			# Single line comment should begin with a space.
```

Comment:
This one is good.

Didn't find one about starting with an uppercase and ending with a dot.